client/src/components/UsedOnTable/UsedOnTable.scss

```diff
@@ -1,16 +1,25 @@
+.used-on__row {
+  cursor: pointer;
+}
+
 .used-on__badge {
   border-color: #cf3f00;
   color: $state-draft;
   background-color: $state-draft-bg;
 }
 
-.used-on__col--index {
+.used-on__index {
```

Comment:
Are you able to make this change without changing the class names? I'd consider this a breaking change, devs may have CSS or JavaScript referencing this class name